pipe/windobserverii_Combined/windobserverii_calibration_group_and_convert.yaml

```diff
@@ -0,0 +1,176 @@
+    Rscript ./flow.cal.conv.R  \
+      DirIn=/tmp/pfs/filter_joined \
+      DirOut=/pfs/out \
+      DirErr=/pfs/out/errored_datums \
+      FileSchmData=$FILE_SCHEMA_DATA \
+      FileSchmQf=$FILE_SCHEMA_FLAGS \
+      "TermQf=u_axis_wind_speed|v_axis_wind_speed"
+      
+    EOF
+  env:
+    # Environment variables for filter-joiner.
+    # There are two configs here, CONFIG_KAFKA is when data comes from kafka, CONFIG_TRINO is when data comes from trino. 
+    # The engironment variable CONFIG is set to the appropriate one as detected in the bash script above
+    CONFIG_TRINO: |
+      ---
+      # Configuration for filter-joiner module that will bring together the data and calibrations
+      # In Pachyderm root will be index 0, 'pfs' index 1, and the repo name index 2.
+      # Metadata indices will typically begin at index 3.
+      input_paths:
+        - path:
+            name: DATA_PATH_TRINO
+            # Filter for data directory
+            glob_pattern: /pfs/DATA_PATH_TRINO/windobserverii/*/*/*/*/**
+            # Join on named location (already joined below by day)
+            join_indices: [7]
+            outer_join: true
+        - path:
+            name: CALIBRATION_PATH
+            # Filter for data directory
+            glob_pattern: /pfs/CALIBRATION_PATH/windobserverii/*/*/*/*/**
+            # Join on named location (already joined below by day)
+            join_indices: [7]
+    CONFIG_KAFKA: |
+      ---
+      # Configuration for filter-joiner module that will bring together the data and calibrations
+      # In Pachyderm root will be index 0, 'pfs' index 1, and the repo name index 2.
+      # Metadata indices will typically begin at index 3.
+      input_paths:
+        - path:
+            name: DATA_PATH_KAFKA
+            # Filter for data directory
+            glob_pattern: /tmp/kafka_merged/windobserverii/*/*/*/*/**
+            # Join on named location (already joined below by day)
+            join_indices: [7]
+            outer_join: true
+        - path:
+            name: CALIBRATION_PATH
+            # Filter for data directory
+            glob_pattern: /pfs/CALIBRATION_PATH/windobserverii/*/*/*/*/**
+            # Join on named location (already joined below by day)
+            join_indices: [7]
+    OUT_PATH: /tmp/pfs/filter_joined # Note that R modules use "pfs" in the path structure to determine datums
+    LOG_LEVEL: INFO
+    RELATIVE_PATH_INDEX: "3" # Must be consistent across inputs 
+    LINK_TYPE: COPY # options are COPY or SYMLINK. MUST BE SIMLINK IF USING COMBINED MODULE.
+    # Environment variables for calibration module
+    PARALLELIZATION_INTERNAL: '3' # Option for calibration conversion module
+input:
+  cross:
+  - pfs:
+      name: FILE_SCHEMA_L0
+      repo: windobserverii_avro_schemas
+      glob: /windobserverii/windobserverii.avsc
+  - pfs:
+      name: FILE_SCHEMA_DATA
+      repo: windobserverii_avro_schemas
+      glob: /windobserverii/windobserverii_calibrated.avsc
```

Comment:
There is no calibrated output for this source type. Please delete the FILE_SCHEMA_DATA input from the pipeline specification.